analysis_options.yaml

```diff
@@ -1,8 +1,8 @@
 include: package:lints/recommended.yaml
 
 analyzer:
-  strong-mode:
-    implicit-casts: false
+  language:
+    strict-casts: true
```

Comment:
I'd assume that a `2.12.0` analyzer will permit this (won't give an error) but that strict-casts may not be enabled?

Reply:
I think a Dart 2.12.0 analyzer actually reports an error. See: https://github.com/dart-lang/timing/actions/runs/3877844370

Reply:
This package has as its min sdk 2.12.0. It looks like we're only testing against `dev` though.

Do you know what sdk is the earliest where you could add this option w/o an analysis error?

Reply:
I don't know the earliest version that `strict-casts` is available in. Why do you ask? Do you want to add a stable channel task for GitHub Actions?

Reply:
On several of our bots we test the oldest supported sdk as well as stable or dev. It would be nice to know what value to set the min sdk to where analysis would still pass.

Reply:
I didn't write down in sdk's CHANGELOG when strict-casts was introduced. ☹️  I'll look it up.

Reply:
It was added in November 2021, maybe Dart 2.15?

Reply:
I don't think it's a big deal. It doesn't affect any runtime behavior. As long as folks editing this code are on latest, I'd say punt and land this!